src/pages/warranty/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { GetStaticProps } from 'next';
+import { Container, Typography } from '../../components/dist';
+import { WarrantyForm } from '../../components';
+import { fetchContact } from '../../lib';
+import { WarrantyPage } from '../../types';
+
+export interface WarrantyProps {
+    warrantyPage: WarrantyPage;
+}
+
+export const getStaticProps: GetStaticProps = async (): Promise<{props: WarrantyProps;}> => {
+    const warrantyPage = await fetchContact();
+    return {
+        props: {
+            warrantyPage
+        }
+    };
+};
+  
+export default function Warranty(props: WarrantyProps) {
+    const { warrantyPage } = props;
+    const { form, inputs, title } = warrantyPage;
+
+    return (
+        <>
+            <Container sx={{ mt: 12 }}>
+                <Typography paragraph variant="h1">
+                    Warranty
+                </Typography>
+                <WarrantyForm form={form} inputs={inputs} />
+            </Container>
+        </>
+    );
+}
```

Comment:
missing new line

Reply:
The frontend code was lot of copy paste, will remove it in updated code as I had to do for contact us too.